scripts/main.js

```diff
@@ -0,0 +1,57 @@
+let books;
+// eslint-disable-next-line
+if (localStorage.getItem('books') === null) {
+  books = [];
+} else {
+  // eslint-disable-next-line
+  books = JSON.parse(localStorage.getItem('books'))
+}
+
+document.querySelector('form').addEventListener('submit', () => {
+  const bookTitle = document.querySelector('.title').value;
+  const bookAuthor = document.querySelector('.author').value;
+
+  const book = [{ bookTitle, bookAuthor }];
```

Comment:
[OPTIONAL] Consider adding a unique id to your books.

<img width="852" alt="Screen Shot 2022-08-02 at 7 54 29" src="https://user-images.githubusercontent.com/82426682/182379628-7ebd042f-ac4d-418b-9b67-49da2ecc1718.png">